sections/featured-product.liquid

```diff
@@ -681,6 +684,12 @@
   </script>
   <script src="{{ 'product-model.js' | asset_url }}" defer></script>
 {%- endif -%}
+{% if section.settings.image_zoom == 'hover' %}
+  <script id="enableZoomOnHover-featured" src="{{ 'magnify.js' | asset_url }}" defer="defer"></script>
+{% endif %}
```

Comment:
I also wonder... if I visit the store on mobile only I'm getting a network request for nothing right ? I wonder what's best between: 
- having the JS in `global.js` and have a `matchMedia()` check to run the function based on `hover` as well (`const canHover = window.matchMedia('(hover: hover)').matches; //true or false`)
- keep it as is and have an network request that isn't necessary as well as the code within the file for devices that don't support `hover`.

The difference is probably minimal 🤷

Reply:
I previously considered [using js to determine whether we're on a mobile device](https://developer.mozilla.org/en-US/docs/Web/HTTP/Browser_detection_using_the_user_agent), but during a pairing session decided to go with CSS at the time. I'm starting to think that approach is causing some inconsistencies, and it might be better to just use javascript for this.